### PR TITLE
XRT-1404: Switched from using numerical strings to actual integers and floats for resource attributes

### PR DESCRIPTION
**Pull request by FraserG01**
Also updated the example profiles to include a larger array of example resources

### CODE REVIEW

**Comment by FraserG01** [comment]
Not ready for review yet actually as CI is failing for XRT

File: AzureSphere/config/profiles/device-virtual.json [hunk 1]
```diff
@@ -8,285 +9,140 @@
...
-          "readWrite": "RW"
-        },
-        "units": {
-          "type": "String",
-          "readWrite": "R",
-          "defaultValue": "random bool value"
-        }
-      }
+      "name": "RandomUInt64",
+      "description": "Generate random UInt64 value. Missing maximum attribute as integer attributes are read in as Int64 values. Will let it default instead",
```
[the remark this answers is not in the document]
Comment (iain-anderson) [reply AzureSphere/config/profiles/device-virtual.json:53]: @FraserG01 maybe best to remove all of the min/max attributes. Only put them in if we want an example of constraining the range

[the remark this answers is not in the document]
Comment (FraserG01) [reply AzureSphere/config/profiles/device-virtual.json]: I agree, have removed limits on standard resources. I have also added some extra resources with limits

**Comment by FraserG01** [comment]
ready for review @iain-anderson

**Review by iain-anderson — APPROVED** [review]
LGTM

**Pull request merged**